README.md

```diff
@@ -33,6 +33,11 @@ Please ***INSTALL [Git for Windows](https://git-scm.com/download/win)*** and run
     0. Click <img src="http://gtms04.alicdn.com/tps/i4/TB1wCcqMpXXXXakXpXX3G7tGXXX-34-44.png" height="16" > (`Run` button)
 0. [Add an example](./examples/README.md#add-an-example)
 
+
+#### Runtime
+
+On Android Platform , Weex code executed by [weex_v8core](https://github.com/alibaba/weex_v8core) which based on google v8 JavaScript engine.
```

Comment:
On Android platform, Weex code is executed in [weex_v8core](https://github.com/alibaba/weex_v8core) which is based on Google V8 JavaScript engine.